diff --git a/stupidterm.c b/stupidterm.c
index e8155fa..dd79b93 100644
--- a/stupidterm.c
+++ b/stupidterm.c
@@ -687,6 +687,7 @@ setup(int argc, char *argv[])
 	vte_terminal_set_mouse_autohide(terminal, conf.mouse_autohide);
 	vte_terminal_set_cursor_blink_mode(terminal, VTE_CURSOR_BLINK_OFF);
 	vte_terminal_set_cursor_shape(terminal, VTE_CURSOR_SHAPE_BLOCK);
+        vte_terminal_set_bold_is_bright(terminal, TRUE);
 	if (conf.lines)
 		vte_terminal_set_scrollback_lines(terminal, conf.lines);
 	if (conf.palette_size) {
